srtcore/buffer.cpp

```diff
@@ -63,6 +63,41 @@ using namespace std;
 using namespace srt_logging;
 using namespace srt::sync;
 
+bool AvgBufSize::canUpdate(const time_point& now) const
```

Comment:
I think the name better suggests what is the content of the condition. Maybe something like `timeToProbe` or `samplingPeriodElapsed`?

Reply:
Changed to `isTimeToUpdate()`